### DISCUSSION TITLE
mmdeploy inference code for mmediting

### DISCUSSION THREAD

**Discussion by MuGeeee** [open-mmlab/mmdeploy#1462] [Show and tell]
At present, there is no c++ inference code for the deployed mmediting model, there is a problem with the code I parodied mmdetection
，I hope someone can give me a correct code, just the core part of inference, thank you，below is my wrong  code
`

    string model_path = "../mmdeploy_model/realesrnet";
    mmdeploy::Model model(model_path.c_str());

    mmdeploy::Restorer restorer(model, mmdeploy::Device{device_name, device_id});
    startTime = clock();
    for (int j = 0; j < filesVec.size(); j++)
    {
        Mat img = imread(filesVec[j], 1);
        Mat img2;
        img.convertTo(img2, CV_32FC3);
        auto rets = restorer.Apply(img2);`

**Comment by lvhan028** [comment]
This is the C++ API for mmediting super-resolution model inference: https://github.com/open-mmlab/mmdeploy/blob/master/csrc/mmdeploy/apis/cxx/mmdeploy/restorer.hpp 

You can also read its demo code for the specific usage: https://github.com/open-mmlab/mmdeploy/blob/master/demo/csrc/cpp/restorer.cxx

**Reply by MuGeeee** [reply]
thanks a lot, you do help me really!
